Makefile

```diff
@@ -159,9 +160,18 @@ update-helm-version:
 	$(MAKE) update-helm-version-access-slack
 	$(MAKE) update-helm-version-access-pagerduty
 	$(MAKE) update-helm-version-access-mattermost
+	$(MAKE) update-helm-version-plugin-event-handler
 
 # Update specific chart
-.PHONY: update-helm-version-%
+.PHONY: update-helm-version-plugin-%
+update-helm-version-plugin-%:
```

Comment:
It's effectively a duplicate of ```update-helm-version-access```, why not just merge the 2 targets together to avoid the duplication?